### PR TITLE
zimwriterfs is now a part of zim-tools

### PR DESCRIPTION
**Pull request by asashnov**
Related to https://github.com/openzim/zim-tools/issues/92

Now 'zimwriterfs' is a part of 'zim-tools', not standalone project.

### CODE REVIEW

**Comment by asashnov** [comment]
CI fails because it tries to run ```kiwix-build zimwriterfs``` but we removed this target by this PR.
Configuration of CI is outside from the sources, so I can't fix it myself.

**Comment by asashnov** [comment]
CI configuration is also fixed in .github directory

**Comment by kelson42** [comment]
@asashnov Any progress here? The PR is still in draft?

File: kiwixbuild/dependencies/zim_tools.py [hunk 1]
```diff
@@ -16,6 +16,9 @@ class Builder(MesonBuilder):
 
         @property
         def configure_option(self):
+            base_option  = ""
+            if self.buildEnv.platformInfo.build not in ('android', 'win32'):
+                base_option += " -Dmagic-install-prefix={buildEnv.install_dir}"
```
Comment (mgautierfr) [inline kiwixbuild/dependencies/zim_tools.py:21]: Do we really need to test for android ? We don't build `zim_tools` on android.

File: kiwixbuild/dependencies/all_dependencies.py [hunk 2]
```diff
@@ -13,10 +13,10 @@ class Builder(NoopBuilder):
...
             base_deps = ['zlib', 'lzma', 'zstd', 'xapian-core', 'pugixml', 'libcurl', 'icu4c', 'mustache', 'libmicrohttpd']
-            # zimwriterfs
+            # zimwriterfs from zim-tools
             if platformInfo.build not in ('android', 'win32'):
                 base_deps += ['libmagic', 'gumbo']
-            
+
             # zimtools
             if platformInfo.build not in ('android','iOS'):
                 base_deps += ['docoptcpp']
```
Comment (mgautierfr) [inline kiwixbuild/dependencies/all_dependencies.py:22]: We should "merge" this tests.
zimwriterfs is part of zimtools. We should only have a specific case for win32 as we don't compile it on windows.

**Comment by kelson42** [comment]
@asashnov I have rebased. Please take in consideration last comments by @mgautierfr

**Comment by mgautierfr** [comment]
I've fixed the last issues about the dependencies as we need this PR asap to compile zim-tools.

**Comment by kelson42** [comment]
@mgautierfr Thank you very much for having patched this PR

**Pull request merged**